from builtins import object
class ErrorHandler(object):
    def __init__(self, config):
        self.config = config

    def handle_error(context, handler, exception):
        pass
